.github/ISSUE_TEMPLATE/feature.yml

```
@@ -1,100 +1,87 @@
+      description: Any other context, screenshots, or relevant information
+      placeholder: "Other details that might help with implementation..."
+    validations:
+      required: false
+  - type: input
+    id: meme-upload
+    attributes:
+      label: Mandatory Meme Upload
+      description: Please upload a relevant (or completely random) meme
+      placeholder: "Drag and drop your meme image here or click to upload"
+    validations:
+      required: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)